components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -0,0 +1,173 @@
+$white:#ffffff;
+$black:#000000;
+$light-gray:#aaaaaa;
+$dark-gray:#2e2e2e;
+$gold:#c28e0e;
+
+.intro {
+	background-color: #c28e0e;
+	padding: 1.5em;
+
+
+	&__title {	
+	font-size: 2em;
+	line-height: 1.3em;
+	color: $black;
+	text-align: center;
+	text-transform:uppercase;
+	padding: .5em 0;
+	}
+	
+	&__issue {
+	font-size: 1em;
+	line-height: 1em;
+	color: $black;
+	text-align: center;
+	padding: 0;
+	}
+	
+	&__issue::after {
+	content:" ";
+	display: block;
+	width: 30%;
```

Comment:
Should we make this a hardcoded width instead of percentage-based?

Reply:
I would say this would be a question for Garrett. For the time being, let's leave it as a note and let him comment when it's time for him to review.